spec/mailers/candidate_mailer_spec.rb

```diff
@@ -14,9 +14,20 @@
   let(:candidate) { build_stubbed(:candidate) }
   let(:application_choices) { [build_stubbed(:application_choice)] }
   let(:dbd_application) { build_stubbed(:application_choice, :dbd) }
+  let(:interview) do
+    create(:interview,
+           date_and_time: Time.zone.local(2021, 1, 15, 9, 30),
+           location: 'Hogwarts Castle',
+           additional_details: 'Bring your magic wand for the spells test')
+  end
+  let(:application_choice_with_interview) { interview.application_choice }
```

Comment:
I think to make it easier to see this setup we should create a new context for all interview related specs and move this setup in it.